app/src/main/java/com/edu/sun/oereminder/utils/DateTimeExtension.kt

```diff
@@ -0,0 +1,29 @@
+package com.edu.sun.oereminder.utils
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+const val DATE_FORMAT = "dd MMMM"
+const val DATE_FORMAT_DAY_OF_WEEK = "EEEE, dd-MM-yyyy"
+const val TIME_FORMAT = "hh:mm"
+const val TIME_NULL = "--:--"
+
+fun GregorianCalendar.toDate(): String =
+    SimpleDateFormat(DATE_FORMAT_DAY_OF_WEEK, Locale.getDefault()).format(time)
+
```

Comment:
```suggestion
fun GregorianCalendar.toDate() =
    SimpleDateFormat(DATE_FORMAT_DAY_OF_WEEK, Locale.getDefault()).format(time)

```

Reply:
Trường hợp ngày ko được
"which can lead to unchecked nullability issues"